packages/hash/shared/src/types.ts

```diff
@@ -250,6 +254,10 @@ const workspaceLinkTypes = {
     title: "Author",
     description: "The author of something.",
   },
+  admin: {
+    title: "Admin",
+    description: "The admin of something.",
```

Comment:
Just so I'm on the same page, we are using it as `workspaceInstance -admin-> user`, right?

Reply:
Correct 👍